backend/src/test/java/com/github/liliancharlotte/controller/UserControllerTest.java

```diff
@@ -0,0 +1,89 @@
+package com.github.liliancharlotte.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.liliancharlotte.model.JobPosting;
+import com.github.liliancharlotte.model.User;
+import com.github.liliancharlotte.repository.UserRepo;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+
+import java.util.ArrayList;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@AutoConfigureMockMvc
+@SpringBootTest
+class UserControllerTest {
+
+    @Autowired
+    MockMvc mockMvc;
+
+    @Autowired
+    UserRepo userRepo;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    String userEndPoint = "/api/user";
```

Comment:
Wenn sich der String nicht ändert, mache das Feld gerne `private static final` um das sichtbar zu machen und auch für die Zukunft auszuschließen.
```suggestion
    private static final String userEndPoint = "/api/user";
```

Reply:
Good point!